elytron/WFCORE-5170-multiple-certificate-revocation-lists.adoc

```diff
@@ -0,0 +1,189 @@
+= [WFCORE-5170] Certificate Revocation Lists
+:author:            Sonia Zaldana Calles
+:email:             szaldana@redhat.com
+:toc:               left
+:icons:             font
+:idprefix:
+:idseparator:       -
+
+== Overview
+
+Certificate Revocation Lists (CRLs) contain a list of certificates that have been revoked
+by the issuing Certificate Authority before their expected expiration date, and therefore should no longer
+be trusted. Elytron currently supports configuring one Certificate Revocation List.
+
+However, if several Certificate Authorities are used, there is no way to configure several
+CRL files. This task is to enable support for more than one Certificate Revocation List in Elytron.
+
+== Issue Metadata
+
+=== Issue
+
+* https://issues.redhat.com/browse/WFCORE-5170[WFCORE-5170]
+
+=== Related Issues
+
+* https://issues.redhat.com/browse/WFLY-14007[WFLY-14007]
+* https://issues.redhat.com/browse/ELY-2033[ELY-2033]
+* https://issues.jboss.org/browse/EAP7-1548[EAP7-1548]
+
+
+=== Dev Contacts
+
+* mailto:{email}[{author}]
+
+=== QE Contacts
+
+TBD
+
+=== Testing By
+// Put an x in the relevant field to indicate if testing will be done by Engineering or QE.
+// Discuss with QE during the Kickoff state to decide this
+* [ ] Engineering
```

Comment:
@SoniaZaldana Since this was tested by Engineering, this section can now be updated. Thanks.